Docs/source/usage/examples.rst

```diff
@@ -60,6 +60,12 @@ Laser-ion acceleration
    The resolution of this 2D case is extremely low by default.
    You will need a computing cluster for adequate resolution of the target density, see comments in the input file.
 
+.. warning::
+
+   It is strongly advised to set the parameters ``<species>.zmin / zmax / xmin / ...`` when working with highly dense targets that are limited in one or multiple dimensions.
```

Comment:
@n01r @RevathiJambunathan just as a follow-up to our hackathon activities: We can remove this note once #3392 was merged